### PR TITLE
Client mappings

### PR DESCRIPTION
**Pull request by IotaBread**
Maps things in the `net/minecraft/client` package, mostly stuff in the `realms`, `gui` and `particle` sub-packages

### CODE REVIEW

File: mappings/net/minecraft/client/realms/gui/screen/RealmsScreen.mapping [hunk 1]
```diff
@@ -1,9 +1,35 @@
...
+	FIELD f_chyfprve COMPONENT_HEIGHT I
+	FIELD f_dqlzfoxk LINK_HOVER_COLOR I
 	FIELD f_dtagjwpk labels Ljava/util/List;
+	FIELD f_esmkvhrf SKIN_TEXTURE_HEIGHT I
+	FIELD f_itiwrdzw TITLE_HEIGHT I
+	FIELD f_ivchwtfm DARK_GRAY I
+	FIELD f_jkrftkld BRIGHT_YELLOW I
+	FIELD f_lavhartt LINK_COLOR I
+	FIELD f_ldlqzcrm GRAY I
+	FIELD f_mggdlalg EXPIRATION_NOTIFICATION_DAYS I
```
Comment (triphora) [inline mappings/net/minecraft/client/realms/gui/screen/RealmsScreen.mapping:17]: Might this be a better name?
```suggestion
	FIELD f_mggdlalg DAYS_UNTIL_NOTIFICATION_EXPIRATION I
```

File: mappings/net/minecraft/client/realms/dto/RealmsWorldOptions.mapping [hunk 2]
```diff
@@ -2,18 +2,26 @@ CLASS net/minecraft/unmapped/C_wbvlbazq net/minecraft/client/realms/dto/RealmsWo
...
+	FIELD f_hjxcdrii SPAWN_NPCS_DEFAULT Z
 	FIELD f_hwzwddmo slotName Ljava/lang/String;
 	FIELD f_hxioygrg spawnAnimals Z
 	FIELD f_hyuoljdz templateId J
+	FIELD f_icfqvkka PVP_DEFAULT Z
 	FIELD f_jewdoawb gameMode I
+	FIELD f_jiyvasuw SPAWN_PROTECTION_DEFAULT I
 	FIELD f_kdbebnoe templateImage Ljava/lang/String;
+	FIELD f_kxysqevw ENABLE_COMMAND_BLOCKS_DEFAULT Z
+	FIELD f_laxynhpb SPAWN_MOSTERS_DEFAULT Z
```
Comment (triphora) [inline mappings/net/minecraft/client/realms/dto/RealmsWorldOptions.mapping]:
```suggestion
	FIELD f_laxynhpb SPAWN_MONSTERS_DEFAULT Z
```

**Review by triphora — APPROVED** [review]

File: mappings/net/minecraft/client/realms/gui/screen/RealmsScreen.mapping (hunk 1 above)
Comment (IotaBread) [reply mappings/net/minecraft/client/realms/gui/screen/RealmsScreen.mapping:17]: Iirc it's the number of days before expiration for the notification to appear

Comment (triphora) [reply mappings/net/minecraft/client/realms/gui/screen/RealmsScreen.mapping:17]: Oh, oops

**Review by OroArmor — APPROVED** [review]

**Review by LambdAurora — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]

**Pull request merged**